README.md

```diff
@@ -22,49 +34,149 @@ You can do it manually following the instructions from [rust-lang.org](https://w
+2. After all the preparations you can launch a node:
+* to start exe`$~/.../<path_to_node.exe>`
+*  also you need to link working directory, which will be a storage for all node associated files. It can be done with `--repo` flag
+*  One more important linking is configuration and genesis files, you can pass paths to them with `--config <path>` and `--genesis <path>` respectively.
+*  For specifing peers listening port you need to pass `--port <port>` flag
+*  In case you want to make a node + miner configuration or multi-node configuration, you need to open api-port. You can do that with `--api <port>`
+*  Example of the comand: `$~/.../<path_to_node.exe> --repo REPO --genesis genesis.car --config config.cfg --port 8080 -- api 1235`
+
+3. Once the node has been successfully configured, it will start to connect to network peers and download blockchain history. This procces may continue for a while(f.e. the weight of chain of an enteropnet network is 3.5 GB). As it takes a while, you can change cids_index.car file to downloaded snapshot(see step 1 for that).
+
+## Lotus-CLI establishing
+### User approach:
+1. Download [lotus-CLI executable](https://github.com/filecoin-project/lotus/releases/ "lotus releases")
+2. You can use CLI client with your node by linking it with `--repo <path to your node working directory>` flag,
+   more of lotus-CLI flags will be covered in **Work with Lotus-CLI**
+
+### Advananced approach:
+1. clone [lotus repository](https://github.com/filecoin-project/lotus "lotus project")
+2. build lotus cli client with command: `make build`
+3. In case you want connect to interopnet, build lotus with `make interopnet`
+## Work with Lotus-CLI
+
+1. For proper miner work you will need a bls wallet, which can be made with lotus-cli by using the following command: `$lotus --repo <repo of your working node> wallet new bls`, after the execution of this comand you will revcieve the public address of your new account, and a private key in `<repo of youe working node>/keystore`.
+2. Also you may need to put some Filecoins on your account from other addresses you may have, and you can do it  with `lotus --repo <repo of your working node> send --from <sender address> <public address of your account>`
+
+3. If you dont have any Filecoins or you want to get more, you can buy them on stocks with usage of you public address.
+
+4. Other advanced commands you may find in [filecoin-docs about lotus CLI](https://docs.filecoin.io/mine/lotus/dagstore/#cli-commands "lotus CLI documentation").
+## Fuhon miner establishing
+To start mining at first you need to configure and run Fuhon, Lotus, Venus or any other filecoin node. Before all make sure that your node had successfully **connected** and updated to the current **block height** (actual network block number). Also, your node's api port should be opened for conection.
```

Comment:
"Venus or any other filecoin node" is misleading.
this doc describes fuhon-miner and lotus-miner, which relies on lotus compatible rpc.

Reply:
fixed